.github/workflows/ci.yml

```diff
@@ -21,9 +21,15 @@ on:  # yamllint disable-line rule:truthy
   schedule:
     - cron: '28 1,7,13,19 * * *'
   push:
-    branches: ['v[0-9]+-[0-9]+-test', 'providers-*/v[0-9]+-[0-9]+]']
+    branches:
```

Comment:
Should we include quotes for safe YAML parsing

Reply:
We had discussion about it with @jscheffl recently - I think it does not change safety, and while I favoured quoting, I   see how a bit nicer it is when you can skip them.

@jscheffl -> consider I've learned something new :)

Reply:
YAML is made to be less boilerplate. No quotes needed except if you want to force bool/int to be string. I favor w/o quotes. But might be a matter of taste. No difference in security.

Reply:
Ohh, thats good.
Thanks for the insights!

Reply:
Mhm, but reading the YAML specs in https://yaml.org/spec/1.2.2/#plain-style I _think_ @shubhamraj-git is right. If the literals `[` and `]` arecontained then it is advised to quote it:

> In addition, inside [flow collections](https://yaml.org/spec/1.2.2/#flow-collection-styles), or when used as [implicit keys](https://yaml.org/spec/1.2.2/#example-single-pair-explicit-entry), plain scalars must not contain the “[”, “]”, “{”, “}” and “,” characters.

Reply:
Well. Just to have some interesting discussion on specs:

> Mhm, but reading the YAML specs in https://yaml.org/spec/1.2.2/#plain-style I think @shubhamraj-git is right. If the literals [ and ] arecontained then it is advised to quote it:

> In addition, inside [flow collections](https://yaml.org/spec/1.2.2/#flow-collection-styles), or when used as [implicit keys](https://yaml.org/spec/1.2.2/#example-single-pair-explicit-entry), plain scalars must not contain the “[”, “]”, “{”, “}” and “,” characters.

* Yeah. "flow collections" was what it was before (and it was quoted) - but I just changed it to `block collections' :)
* Implicit keys - are different things.

So we are safe :)